src/rtctools_heat_network/workflows/grow_workflow.py

```diff
@@ -175,7 +175,8 @@ def read(self):
                 else:
                     total_demand += demand_values
                 max_demand = max(demand_values)
-                self.__heat_demand_bounds[f"{demand}.Heat_demand"] = (0.0, 2 * max_demand)
+                # TODO: check why (0.0, 2.0 * max_demand) caused investment to go to upper bound
+                self.__heat_demand_bounds[f"{demand}.Heat_demand"] = (0.0, max_demand)
```

Comment:
let's revert this change as it is related to how the constraints are setup and make a seperate MR for fixing this.

Reply:
Agreed. We need this fix before PoC release though

Reply:
Code has been reverted back to the original code